src/client.ts

```diff
@@ -7992,9 +7992,9 @@ export class MatrixClient extends EventEmitter {
         limit?: number,
         batch?: string,
     ): Promise<{
-        rooms: ISpaceSummaryRoom[];
-        events: ISpaceSummaryEvent[];
-    }> {
+            rooms: ISpaceSummaryRoom[];
+            events: ISpaceSummaryEvent[];
+        }> {
```

Comment:
The indentation seems a little weird here :thinking:

Reply:
/me is setting up prettier while being very angry

Reply:
Ah :grimacing:

Reply:
It's due to eslint rules.